src/main/java/stream/flarebot/flarebot/commands/management/PermissionsCommand.java

```diff
@@ -359,7 +375,9 @@ public String getUsage() {
                 "`{%}permissions group <group> list [page]` - lists the permissions this group has.\n" +
                 "`{%}permissions group <group> massadd <@everyone/@here/role>` - Puts everyone with the giving role into the group.\n" +
                 "`{%}permissions group <group> clear` - Removes all permissions from this group!\n" +
-                "`{%}permissions group <group> move <pos>` - Moves the group to a different position on the hierarchy.\n\n" +
+                "`{%}permissions group <group> move <pos>` - Moves the group to a different position on the hierarchy.\n" +
+                "`{%}permissions group <group> clone <new_group>` - clones a group\n" +
+                "`{%}permissions group <group> rename <new_name>` - renames the group\n\n" +
```

Comment:
Start the descriptions with capitals please

Reply:
Also add full stops

Reply:
:+1: